test/functionalTest/cases/2248_forwarding_updates_overwrite_the_types_of_attributes/forwarding_updates_overwrite_the_types_of_attributes.test

```diff
@@ -139,12 +141,18 @@ Date: REGEX(.*)
 
 
 03. Update E1/T1/A1 in CB, using PUT /v2/entities/E1/attrs/A1/value, without stating type of the attribute, value is a string
+Note: we get 404 Not Found, but is ok. It is due to accumulator-server.py doesn't answers in NGSIv1
 =============================================================================================================================
-HTTP/1.1 204 No Content
-Content-Length: 0
+HTTP/1.1 404 Not Found
+Content-Length: 61
+Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
+{
```

Comment:
Case 1) seems to be covered by https://github.com/telefonicaid/fiware-orion/blob/release/1.2.0/test/functionalTest/cases/0787_cprs_full_functional/update_false_registration_in_ngsiv2.test (note the *release/1.2.0* in the URL).